src/sentry/utils/mockdata/core.py

```diff
@@ -1238,6 +1245,79 @@ def generate_performance_issues():
         generate_performance_issues()
 
 
+def create_mock_attachment(event_id, project):
+    attachment_id = str(uuid.uuid4())
+    file_dir = os.path.dirname(os.path.abspath(__file__))
+    attachment_path = os.path.join(file_dir, "../../static/sentry/images/favicon.png")
```

Comment:
A little hacky, but 1) didn't want to upload a new image and 2) can't assume `sentry` dir is always at `~`